src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -182,13 +202,61 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             // Add disabled versions
             AddBlockedVersions(blockedVersions);
 
+            // Update the view and add the filter
+            VersionsView = new CollectionViewSource() { Source = Versions }.View;
+            VersionsView.Filter += VersionsFilter;
+
             SelectVersion();
 
             OnPropertyChanged(nameof(Versions));
+            OnPropertyChanged(nameof(VersionsView)); // This can change the selected version if a version is added after SelectedVersion is set
```

Comment:
You resolved my comment about this, so does that mean you found a reason this is needed? I'm curious why this property has to be raised?

Reply:
It doesnt works if I dont add this line, as far as I know this line is needed because we change the source of the View in the same funciton